unlynx/services/api.go

```diff
@@ -51,6 +52,7 @@ func (c *API) SendSurveyDDTRequestTerms(entities *onet.Roster, surveyID SurveyID
 	}
 
 	resp := ResultDDT{}
+	fmt.Println(c.entryPoint.Address)
```

Comment:
Was this used for debugging? If so it should be removed, otherwise use the logger used in the rest of medco-unlynx.

Reply:
yep, forgot